src/pages/BalanceNew/index.tsx

```diff
@@ -718,12 +711,44 @@ const Balance: React.FC<BalanceProps> = () => {
 
   const navigate = useNavigate();
 
+  // Move oraib2oraichain
+  const [moveOraib2OraiLoading, setMoveOraib2OraiLoading] = useState(false);
+  const { remainingOraib } = useGetOraiBridgeBalances(moveOraib2OraiLoading);
+
+  const moveOraibToOraichain = async () => {
```

Comment:
This function should be put outside of the tsx file to write test cases. It should return the list of from, to & transfer amount. Then, we have another function that calls moveOraibToOraichain, then probably call await transferIbcCustom